.travis.yml

```diff
@@ -26,7 +26,7 @@ matrix:
       sudo: required
       dist: trusty
     # As above, xspec 12.9.1n
-    - env: XSPECVER="12.9.1" NUMPYVER=1.13 FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER=2.0 TRAVIS_PYTHON_VERSION="3.6"
+    - env: XSPECVER="12.9.1" NUMPYVER="1.13.3=py36ha12f23b_0" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER=2.0 TRAVIS_PYTHON_VERSION="3.6"
```

Comment:
Perhaps there should be a comment added here to explain the reason for the pinning (just pointing to an issue/comment on GitHub is enough, I think).